benchmark_test.go

```diff
@@ -1,105 +0,0 @@
-package containerd
```

Comment:
I actually think it should be left, I'd like at one point to be able to run them in CI and compare the results with previous run automatically.

If you really want it removed,  can you make it a single commit, so it can easily be reverted later?